src/chart.zoom.js

```diff
@@ -159,7 +172,7 @@ function doPan(chartInstance, deltaX, deltaY) {
 
 		helpers.each(chartInstance.scales, function(scale, id) {
 			if (scale.isHorizontal() && directionEnabled(panMode, 'x') && deltaX !== 0) {
-				panScale(scale, deltaX);
+				panScale(scale, deltaX, helpers.getValueOrDefault(chartInstance.options, defaultOptions));
```

Comment:
I think its best to either pass `chartInstance.options` or `panOptions` here depending on if we want the pan method to have access to all the options or just the pan options. It might make sense to add a check further up that ensures the options are present so that we don't have to have all kinds of these checks everywhere (i'm fine with doing that in a separate pull request)